modules/lib/lib-app/src/main/resources/lib/xp/app.js

```diff
@@ -0,0 +1,60 @@
+var value = params[name]/**
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
var value = params[name];/**
```